test/lib/create-next-install.js

```diff
@@ -183,6 +191,15 @@ async function createNextInstall({
           .traceAsyncFn(() => installDependencies(installDir, tmpDir))
       }
 
+      if (useRspack) {
+        // This is what the @next/plugin-rspack does.
+        // TODO: Load the plugin properly during test
```

Comment:
@timneutkens is there a straightforward way of doing this with the existing test scaffolding code? I tried to create a `next.config.js` to load the plugin but we also declaratively set a static one: https://github.com/vercel/next.js/blob/0c946eb20dea56211bf15940bf406fe82d31c613/test/lib/next-modes/base.ts#L288